src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -772,255 +782,130 @@ static int32_t ConsoleCommandSet(InteractiveConsole& console, const arguments_t&
             }
         }
 
+        std::string var_name = argv[0];
```

Comment:
I should also mention that we don't allow `snake_case` variable names. My apologies for not mentioning it earlier.

Reply:
I've sorted it.